### ISSUE TITLE
[BUG] Pressing button with onclick multiple times may cause command to be run twice

### ISSUE THREAD

**Issue by LeadSeason** [elkowar/eww#851]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

Pressing a button with onclick or onrightclick multiple times in quick succession may cause the associated command to be executed twice. However, when holding the button for a short time, the issue does not occur.

https://github.com/elkowar/eww/assets/60569772/132b4191-181c-4e8e-bbf8-0109e94ffb4d

#### Configuration
eww.yuck
```
(defwindow barbutton
  :monitor 0
  :geometry (geometry
    :x "0%"
    :y "0%"
    :width "100%"
    :height "30px"
    :anchor "top center"
  )
  :staking "overlay"
  :exclusive true
  :reserve (struts
    :distance "40px"
    :side "top"
  )
  :windowtype "dock"
  :wm-ignore false
  (epicbutton)
)


(defwidget epicbutton []
  (box
    (button
      :onclick "notify-send \"deeznuts \$(date +%N)\""
      "yellow"
    )
  )
)
```
eww.scss 
```scss
* {
	all: unset;
}```

### Expected behaviour

The button's command should be executed only once, regardless of how quickly it is clicked.

### Additional context

Distro: ArchLinux 6.4.4-1-cachyos
wayland compositor: swayfx 0.3.2-1
gpudriver: amdgpu
Monitor setup: triple monitor
Installed eww verison: 0.4.0
Installed from aur: [eww-wayland](https://aur.archlinux.org/packages/eww-wayland)

**Comment by elkowar** [comment]
Why should the command only be executed once if you click twice? two clicks, to executions, or am I missing something?

**Comment by heartles** [comment]
I believe the issue is that some clicks in a series of quick clicks trigger the command twice. So, a double click results in the command being triggered 3 times, instead of 2. Assuming I have the same issue as OP at least (also running on wayland, using `eww-wayland` from nixos unstable).

**Comment by LeadSeason** [comment]
Yes clicking twice results in 3 clicks, last one being doubled. 

Noticed while setting swaync and quickly opening and closing caused it to reopen.

**Comment by elkowar** [comment]
Yeaaaa... I fear that's kind of how GTK works, when you click twice, then you get the click, and then the doubleclick... I'll look into if this is preventable
